app/views/tag/gridsEmbed.html.erb

```diff
@@ -0,0 +1 @@
+
```

Comment:
Great, here in this file, you can add this line:

```ruby
      <%= raw auto_link(insert_extras("[notes:%{params[:tagname]}]"), :sanitize => false) %>
```

See how I've put in the text `[notes:_______]` to be substituted in with the `params[:tagname]` variable which you set up in your routes? This will trigger the display of a table of tag related content. 

I know that line is a bit complex, but it's based on this line from the wiki show page: https://github.com/publiclab/plots2/blob/master/app/views/wiki/show.html.erb#L107